build.sh

```diff
@@ -143,8 +143,8 @@ do
 	esac
 done
 
-export PI_GEN=${PI_GEN:-pi-gen}
-export PI_GEN_REPO=${PI_GEN_REPO:-https://github.com/RPi-Distro/pi-gen}
+export BUILD_SCRIPTS="${BUILD_SCRIPTS:-Umbrel OS official build scripts}"
+export BUILD_SCRIPTS_REPO="${BUILD_SCRIPTS:-https://github.com/getumbrel/umbrel-os}"
```

Comment:
I think you meant `BUILD_SCRIPTS_REPO` here

```suggestion
export BUILD_SCRIPTS_REPO="${BUILD_SCRIPTS_REPO:-https://github.com/getumbrel/umbrel-os}"
```

Reply:
Yes, thank you!